docs/source/examples/private_utxo/private_utxo_transaction_family.rst

```diff
@@ -0,0 +1,411 @@
+***************************************
+Private UTXO Transaction Family
+***************************************
+
+Overview
+========
+
+This document describes a Transaction Family that allow for assets to be
+tracked and traded on the Ledger as well as to be held off the Ledger and still
+traded in a manner that enforces the Ledger rules governing the Asset and
+allows transactional privacy for the owner of the asset. Assets in this system
+can exist in two states on-Ledger and off-Ledger. When assets are on-Ledger
+they are held in buckets associated with the owner’s public key. In this state
+the balances and trades of and trades between participants are visible to all
+who can see the ledger. The second state of assets on the Ledger is in the form
+of unspent transaction outputs (UTXO). In this form the assets are represented
+as documents stored off-Ledger and the UTXO are opaque identifiers of these
+documents. Trusted Execution Environments(TEE) are used to validate and process
+off Ledger transactions. Once validated by the TEE, the transaction is
+submitted to the Ledger recording the UTXO consumed and the new UTXO produced.
+
+State
+=====
+
+This section describes in detail how Private UTXO Transaction Processor (PUTP)
+objects are stored and addressed in the global state. All objects in state are
+stored as encoded protobufs. The address of the Merkle nodes is detailed below
+in the addressing section.
+
+AssetType
+---------
+
+AssetTypes represent types of assets being tracked and traded on the Ledger.
+
+.. literalinclude:: ../../../../families/private_utxo/protos/asset_type.proto
+    :language: protobuf
+    :lines: 18-
+
+Holding
+-------
+
+Holdings represent the amount of asset a participant possesses on the Ledger.
+The state entry for holdings is a container that allows the holdings of
+multiple participants to be stored at the same address, this can happen if
+there is a collision in the transformation of the participant public key to the
+address it is stored at in global state. The Holding container is a list of
+participant Holdings, each holding is identified by the participant's public
+key.
+
+The participant Holding is a list of key value pairs, with the key being the
+AssetType address and the value being the amount of the asset the participant
+has.
+
+.. literalinclude:: ../../../../families/private_utxo/protos/holding.proto
+    :language: protobuf
+    :lines: 22-
+
+UTXO
+----
+
+UTXO are stored as empty objects in state. The key uniquely identifies the
```

Comment:
I'm still not sure this is the best idea.  I don't know if it is easy to differentiate between "set but empty" and "not set" in protobuf.  Minimally, the UTXO hash could be stored at the address.